h/accounts/__init__.py

```diff
@@ -96,3 +100,8 @@ def includeme(config):
     config.include('.schemas')
     config.include('.subscribers')
     config.include('.views')
+
+    secret = config.registry.settings['secret_key']
+    derived = derive_key(secret, b'h.accounts')
+    serializer = URLSafeTimedSerializer(derived)
+    config.registry.password_reset_serializer = serializer
```

Comment:
We try not to modify the config registry at runtime, instead add a method to the request object, for example see <https://github.com/hypothesis/h/blob/master/h/accounts/__init__.py#L92> (or grep for `add_request_method`)